.github/workflows/test_conda_store_server_integration.yaml

```diff
@@ -91,6 +88,14 @@ jobs:
           python -m pytest -m "user_journey"
         if: matrix.test-type == 'user-journey'
 
+      # These mirror the tests run while building the conda package
+      # See https://github.com/conda-forge/conda-store-feedstock/ for details
+      - name: "Run basic import tests ✅"
+        run: |
+          python -c "import conda_store_server"
+          conda-store-server --help
+          conda-store-worker --help
+
```

Comment:
We already have a workflow that tests the conda build at https://github.com/conda-incubator/conda-store/blob/main/.github/workflows/conda.yml

Ref run: https://github.com/conda-incubator/conda-store/actions/runs/11789976339/job/32839661575?pr=975

Reply:
we were not aligned with cep25 so we were building with 3.12 only and this did not show up